### ISSUE TITLE
some info missing

### ISSUE THREAD

**Issue by kazola** [ivorivetta/seastate#2]
Hi!
Nice project!
I have a couple questions, though.

1. Which is the geographical area covered by the data? US? Worldwide? 
2. How long back in time goes this data for?

Thanks and have a nice day.

**Comment by kazola** [comment]
Hi!
I was able to dig around and find this :) Sorry, I guess I asked too quick.

The buoy distribution is here ---> https://www.ndbc.noaa.gov/

For the time, you have to ask the API, it will tell you if there is no data.

This project is cool. The only thing is the query's connection fail from time to time but you can manage with a couple try / except.

Bye guys.

**Comment by ivorivetta** [comment]
Thanks for stopping by @kazola ! Sorry these alerts were getting buried in my email but you figured it out. 
The time going back is specific to each buoy and when it went online. I haven't gone back to do any historical studies so don't have that mapped out yet. Now I'm kind of curious and might loop through the bouys to find that out.
I've run into the connection issue too and I've chalked it up to random downtime of the government agency's servers, will look into what I can do to make this library more resilient, if not maybe I should write something in the readme suggesting a try catch when implementing it. At the very least, I'll handle the error gracefully if I didn't already write that when I worked on the code this past summer
